.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**Type of PR**: Bugfix / Feature / Docs / Other
+
+**Required reviews**: 1 / 2 / 3
+
+**What does this PR do? Why is it needed?**
+
+> Try to answer both questions, when possible. 
+> The second one is often not obvious and can help the team to prioritize.
+
+**Which issues(s) does this PR fix?**
+
+> Fixes #...
```

Comment:
```suggestion
**Which issues(s) does this PR fix?**

> Fixes #...

**If applicable, to which feature(s) is this PR contributing?**

> nucypher/productdev/issues/#

```
Of course not all PR's must pertain to a validated feature, but asking this question every time we open a PR will help illuminate what proportion of work each of us is doing that is (1) moving us directly towards adoption, (2) is maintenance or (3) is low-priority or out of sync with team objectives.

Reply:
I agree that having answers to your proposed question and discerning if the PR can be categorized as (1), (2), and (3),  is very important, but I'm not sure what are the best choices to suggest in the template. `nucypher/productdev/issues/` seems a bit limited, and also these options are not mutually exclusive

Reply:
Agreed! I didn't spell out (1), (2) and (3) in the template itself for that reason. As you said in Discord let's leave it as somewhat binary for now – does this PR address a `nucypher/productdev` issue or not, and then improve the template when we have more feature issues in that repo!

Reply:
I'm unsure how to what extent to consider "*If applicable, to which feature(s) is this PR contributing?*".  

Say for example PR #2474 - Do I just say "No"?  In which case, How is this information intended to be used?